webapp/channels/src/components/autosize_textarea.tsx

```diff
@@ -25,10 +25,13 @@ const styles = {
         overflow: 'hidden',
     },
     reference: {
-        display: 'inline-block',
         height: 'auto',
         width: 'auto',
-    },
+        display: 'inline-block',
+        position: 'relative',
+        transform: 'translateY(-100%)',
+        wordBreak: 'break-word'
+    } as CSSProperties,
```

Comment:
Why do we need the casting?

Reply:
I got a typescript error while working on it as it didn't recognize some CSS properties

Reply:
What properties doesn't recognize?

I am wary because casting just "breaks typing", and it may mean that React is expecting something different (or it is just badly typed :sweat_smile:) so I want to make sure it is completely needed here.

Reply:
it doesn't recognise `overflowY: 'auto'` , `position: 'relative'` and  `wordBreak: 'break-word'`

Reply:
Oh! I see the issue now. It is because you are using the strings. You should be able to get rid of the casting if you write with something as this:

```
reference: {
        display: 'inline-block',
        height: 'auto',
        width: 'auto',
        position: 'relative' as const,
        tranform: 'translateY(-100%)',
        wordBreak: 'break-word' as const,
    },
```

Adding the `as const` is another typing, but it is a less dangerous one :P It just tells typescript to not consider this "any string" type, but this particular string, which will match with the types on the CSSProperties.

Would you mind to do the needed changes to remove the `as CSSProperties` you added?

Reply:
Yes I will modify the casting